sirepo/job_api.py

```diff
@@ -63,14 +63,15 @@ def api_downloadDataFile(simulation_type, simulation_id, model, frame, suffix=No
         t = sirepo.job.DATA_FILE_ROOT.join(sirepo.job.unique_key())
         t.mksymlinkto(d, absolute=True)
         try:
-            _request(
+            r = _request(
                 computeJobHash='unused',
                 dataFileKey=t.basename,
                 frame=int(frame),
                 isParallel=False,
                 req_data=req.req_data,
                 suffix=s,
             )
+            assert not r.state == 'error', f'r={r}'
```

Comment:
change 'r='  to 'error state in request='. It's the only thing on the assertion line so it's too terse.

Reply:
https://github.com/radiasoft/sirepo/pull/2609/commits/412a07788212fb1968337d210073456d1ef9152f